### DISCUSSION TITLE
How do I increase the size of lat / long when I issue commands?

### DISCUSSION THREAD

**Discussion by stefanesa** [polymathrobotics/caladan_examples#11] [Q&A] [answered]
I want to more easily find positions to build new commands / behaviors on, but the font size is super small and I'm bad at Python.  Which snippet in the example app should I edit?

**Comment by iliabaranov** [comment, marked as answer]
Hey Stefan!
The font shown on the image is controlled via the font specified at the top of the file:
`font = ("Courier New", 7)`

So just change the 7 above to something larger (say 12) to make it larger.

Cheers!
